frontend/src/pages/ErrorsV2/ErrorBody/components/ErrorBodyText/index.tsx

```diff
@@ -26,12 +26,7 @@ const ErrorBodyText = ({ errorGroup }: Props) => {
 
 	return (
 		<Box display="flex" flexDirection="column" gap="8">
-			<Box
-				py="8"
-				cssClass={style.errorBodyContainer}
-				overflowY="scroll"
```

Comment:
Unrelated but there was a really ugly scrollbar showing:
![Screenshot_2023-06-08_at_9_58_37_AM](https://github.com/highlight/highlight/assets/58678/2959646c-f083-4e9b-a756-0178d0b575f7)

This is unneeded since we truncate for long error bodies with a Show more/Show less button:

![Kapture 2023-06-08 at 10 03 08](https://github.com/highlight/highlight/assets/58678/cbafd9c6-75cc-4571-a7cc-0d286246dbbf)